src/app/api/revalidate/route.ts

```
@@ -1,15 +1,20 @@
 
+/**
+ * @route GET /api/revalidate
+ * @query secret - REVALIDATE_SECRET (env)
+ * @returns `{ revalidated: true, now }` — '/' 경로와 'menu' 태그 강제 재검증
+ */
 export async function GET(request: NextRequest) {
   const secret = request.nextUrl.searchParams.get('secret');
 
   if (secret !== process.env.REVALIDATE_SECRET)
-    return NextResponse.json({message: 'Invalid secret'}, {status: 401});
+    return NextResponse.json({ error: 'Invalid secret' }, { status: 401 });
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)